### ISSUE TITLE
cropArea like a aperture

### ISSUE THREAD

**Issue by ripper2hl** [blueimp/record-screen#1]
https://github.com/wulkano/aperture-node#croparea
![Captura de pantalla de 2019-12-26 20-20-34](https://user-images.githubusercontent.com/2611201/71497305-3bc2b480-281d-11ea-9439-51a5ad5caf1e.png)

This an example of crop with ffmpeg command

`ffmpeg -y -video_size 1920x1080 -r 30 -f x11grab -i :1 -pix_fmt yuv420p -vf crop=960:540:240:450 /tmp/test.mp4`

https://video.stackexchange.com/questions/22000/how-can-i-crop-part-of-a-video-while-recording-from-desktop-in-real-time-using-f

**Comment by blueimp** [comment]
Thanks for the suggestion @ripper2hl.
I've added the new `videoFilter` option and released version `3.12.0`.

You can use the crop filter the following way:

```js
const recording = recordScreen(videoFile, {
  resolution: '1440x900',
  // See https://ffmpeg.org/ffmpeg-filters.html#crop
  videoFilter: 'crop=480:300:960:600'
})
```

**Comment by ripper2hl** [comment]
okay thanks , wtfq ? , really fast xD

this is my code for this feature but you went ahead of me.

```javascript
  if (options.cropArea){
    args.push('-vf', `crop=${options.cropArea.width}:${options.cropArea.height}:${options.cropArea.x}:${options.cropArea.y}`)
  }
```
https://github.com/ripper2hl/record-screen/blob/crop/index.js
